Dockerfile

```diff
@@ -0,0 +1,35 @@
+# FROM node:8-alpine also works here for a smaller image
+FROM node:8-alpine
+
+# set our node environment, either development or production
+# defaults to production, compose overrides this to development on build and run
+ENV NODE_ENV production
+
+EXPOSE 9292 9293
+
+# you'll likely want the latest npm, regardless of node version, for speed and fixes
+# but pin this version for the best stability
+RUN npm i npm@latest -g
```

Comment:
node-alpine already contains "npm". Any reason for installing the latest one here again?

Reply:
It was done to get latest `npm` version, if we want to keep static, which `npm` version shall I set it to?

Reply:
I would assume no explicit version is needed unless we find an issue with the presupplied "npm" in the node-alpine image